### ISSUE TITLE
error adding .git to tar

### ISSUE THREAD

**Issue by ahmetb** [GoogleCloudPlatform/skaffold#160]
I'm getting error from `skaffold dev`:

```
Starting build...
ERRO[0000] Can't add file .git/index to tar: io: read/write on closed pipe
ERRO[0000] Can't close tar writer: io: read/write on closed pipe
```

my config is like

```
apiVersion: skaffold/v1
kind: Config
build:
  tagPolicy: gitCommit
  artifacts:
  - imageName: gcr.io/ahmetb-starter/userdirectory
    workspace: .
    dockerfilePath: Dockerfile.userdirectory
  - imageName: gcr.io/ahmetb-starter/coffeedirectory
    workspace: .
    dockerfilePath: Dockerfile.coffeedirectory
  - imageName: gcr.io/ahmetb-starter/web
    workspace: .
    dockerfilePath: Dockerfile.web
  local: {}
deploy:
  kubectl:
    manifests:
    - paths:
      - misc/kube/
```

and my source code is at https://github.com/ahmetb/coffeelog

I suspect this has something to do with concurrency or something weird about `.git` directory.

**Comment by ahmetb** [comment]
This turned out to be that my `docker` (or `minikube`) wasn't running.

**Comment by r2d4** [comment]
I'm going to close this as something to be implemented in #40
